.changeset/poor-rabbits-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/unenv-preset": patch
+---
+
+Remove clearImmediate and setImmediate injects
```

Comment:
```suggestion
Remove clearImmediate and setImmediate injects.

These are now available in workerd (as of ???? version)
```

Reply:
Since: v1.20240815 - https://github.com/cloudflare/workerd/commit/f07cd8e40f53f1607fb1502916a7fe1f9f2b2862

Reply:
> v1.20240815 

Oh, not so long ago.
Do we want to have an implementation for those function in our preset.
It would be tree shakeable use `setTimeout(0)`, i.e. 1 LOC.